pkg/Cpanel/Security/Advisor/Assessors/Symlinks.pm

```diff
@@ -39,11 +39,33 @@ use base 'Cpanel::Security::Advisor::Assessors';
 
 sub generate_advice {
     my ($self) = @_;
-    $self->_check_for_symlink_kernel_patch;
 
+    if ( $self->_has_cpanel_hardened_kernel() ) {
+        $self->add_warn_advice(
+            'key'  => 'Symlinks_protection_no_longer_support_hardened_kernel',
+            'text' => $self->_lh->maketext('Unsupported cPanel hardened kernel detected.'),
+
+            'suggestion' => $self->_lh->maketext(
+                "[asis,cPanel] no longer supports the hardened kernel. We recommend that you use [asis,KernelCare's] free symlink protection. In order to enable [asis,KernelCare], you must replace the hardened kernel with a standard kernel. For instructions, please read the document on [output,url,_1,How to Manually Remove the cPanel-Provided Hardened Kernel,_2,_3].",
+                'https://go.cpanel.net/uninstallhardenedkernel', 'target', '_blank'
+            ),
+        );
+
+    }
     return 1;
 }
 
+sub _has_cpanel_hardened_kernel {
```

Comment:
If we're going to call this subroutine from outside the module, we'll want to remove the underscore.